README.md

```diff
@@ -85,10 +106,27 @@ atributes := map[string]interface{}{
      "likes_donuts": true,
 }, 
 user := optly.UserContext("optimizely end user", attributes)
-enabled, _ := client.IsFeatureEnabled("binary_feature", user)
+options := []decide.OptimizelyDecideOptions{decide.IncludeReasons}
+decision := userCtx.Decide("my_flag", options)
+if jsonResp, err := json.Marshal(convertDecisionToMap(decision)); err == nil {
```

Comment:
`convertDecisionToMap` is an internal method, end user won't be able to access it. Can we just log the variationKey maybe? or anything that's accessible.